jsettlers.logic/src/jsettlers/logic/movable/Movable.java

```diff
@@ -708,6 +692,13 @@ private void setState(EMovableState newState) {
 		this.state = newState;
 	}
 
+	private void setStrategy(MovableStrategy newStrategy) {
```

Comment:
It is always good to keep common things together. But why did you move only this method?

In my opinion, we can clear this class up even more:
- Separate static and non-static methods
- Group the general attribute getters, order by: id, movable type, attackable, player, state
- Group the whole strategy modification methods
- Group the selection methods (is/set selected)